.drone.star

```diff
@@ -1777,7 +1777,7 @@ def changelog():
                         "push",
                     ],
                     "message": "Automated changelog update [skip ci]",
-                    "branch": "master",
+                    "branch": "stable-2.0",
```

Comment:
If we only publish from the `stable-2.0` or `master` branch respectively, isn't this `ctx.build.source`? 🤔